libs/sam/src/build/get-entries-from-cloudformation.ts

```diff
@@ -42,7 +45,7 @@ function getEntry(
 
     const srcMapInstall = resolve(__dirname, 'source-map-install.js');
     if (resource.Type === 'AWS::Serverless::Function') {
-        return getEntryForFunction(properties, options, srcMapInstall);
+        return getEntryForFunction(properties, options, srcMapInstall, globalProperties.Function);
```

Comment:
`globalProperties` might be undefined here?

Reply:
my bad, made optional now